sdk/schemaregistry/Azure.Data.SchemaRegistry/tests/Samples/Sample02_HelloWorld.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.RegularExpressions;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using NUnit.Framework;
+
+namespace Azure.Data.SchemaRegistry.Tests.Samples
+{
+    public class Sample02_HelloWorld : SamplesBase<SchemaRegistryClientTestEnvironment>
+    {
+#pragma warning disable IDE1006 // Naming Styles
+        private SchemaRegistryClient client;
```

Comment:
Interesting.. why do we need this field?

Reply:
This is so that the different snippets can work off of the same client.